test/e2e/generated_clientset.go

```diff
@@ -324,7 +324,8 @@ var _ = framework.KubeDescribe("Generated release_1_5 clientset", func() {
 		observeCreation(w)
 
 		By("deleting the cronJob")
-		if err := cronJobClient.Delete(cronJob.Name, nil); err != nil {
+		// Use OrphanDependents=false so the delete is synchronous.
+		if err := cronJobClient.Delete(cronJob.Name, &metav1.DeleteOptions{OrphanDependents: new(bool)}); err != nil {
```

Comment:
`OrphanDependents` is being deprecated in 1.7. Should we switch to use `deleteOptions.propagationPolicy` instead?